data/ka'het outfits.txt

```diff
@@ -0,0 +1,194 @@
+	"lifetime" 1
+	"velocity scale" 0
+
+outfit "Ka'het Annihilator"
+	category "Guns"
+	cost 351000
+	thumbnail "outfit/ka'het annihilator"
+	"mass" 10
+	"outfit space" -14
+	"weapon capacity" -14
+	"gun ports" -1
+	weapon
+		sprite "projectile/annihilator"
+			"frame rate" 7
+		sound "plasma"
+		"hit effect" "bullet impact"
+		"inaccuracy" 2.5
+		"velocity" 18
+		"lifetime" 35
+		"reload" 10
+		"firing energy" 31
+		"firing heat" 8
+		"shield damage" 43
+		"hull damage" 12
+	description "The Builders initially created the Annihilator long ago, as a countermeasure to their newly discovered shield matrix. They kept developing it for centuries, with this being its final version."
+
+outfit "Ka'het Annihilator Turret"
+	category "Turrets"
+	cost 529000
+	thumbnail "outfit/ka'het annihilator turret"
+	"mass" 25
+	"outfit space" -32
+	"weapon capacity" -32
+	"turret mounts" -1
+	weapon
+		sprite "projectile/annihilator"
+			"frame rate" 7
+		"hardpoint sprite" "hardpoint/annihilator turret"
+		"hardpoint offset" 15.
+		sound "plasma"
+		"hit effect" "bullet impact"
+		"inaccuracy" 3
+		"turret turn" 2.3
+		"velocity" 18
+		"lifetime" 35
+		"reload" 10
+		"firing energy" 62
+		"firing heat" 16
+		"shield damage" 86
+		"hull damage" 24
+	description "The Builders mounted two Annihilators on this turret, making it highly efficent against enemy shields."
+
+outfit "Ka'het EMP Deployer"
+	category "Secondary Weapons"
+	cost 715000
+	thumbnail "outfit/ka'het emp deployer"
+	"mass" 18
+	"outfit space" -26
+	"weapon capacity" -26
+	"gun ports" -1
+	"emp torpedo capacity" 16
+	weapon
+		sprite "projectile/emp torpedo"
+			"frame rate" 2
+			"no repeat"
+		sound "torpedo"
+		ammo "EMP Torpedo"
+		icon "icon/emp"
+		"fire effect" "emp torpedo fire" 5
+		"hit effect" "nuke explosion"
+		"hit effect" "emp ring" 30
+		"hit effect" "emp spark" 20
+		"die effect" "tiny explosion"
+		stream
+		"reload" 180
+		"firing energy" 540
+		"firing heat" 11
+		"inaccuracy" 4
+		"lifetime" 300
+		"velocity" 7
+		"acceleration" .7
+		"drag" .1
+		"turn" 2
+		"homing" 3
+		"infrared tracking" .7
+		"radar tracking" .8
+		"trigger radius" 50
+		"blast radius" 200
+		"shield damage" 700
+		"hull damage" 100
+		"ion damage" 48
+		"hit force" 90
+		"missile strength" 80
+	description "Eons ago, the Builders were attacked by alien ships which used weapons that ignored their hull and damaged their reactors. Although they never did recover any of these weapons, after years of research they managed to replicate their effects."
+
+# Cooling:
+
+outfit "Ka'het Support Cooling"
+	category "Systems"
+	cost 218000
+	thumbnail "outfit/ka'het support cooling"
+	"mass" 6
+	"outfit space" -8
+	"active cooling" 32
+	"cooling energy" 3.1
+	description "Het ships need a much better cooling than normal ships, partly due to the requirement of keeping the Het alive. While a single Support Cooling isn't enough to sustain a ship, the Ka'het use this module in addition to the stronger Primary Cooling."
```

Comment:
>Het ships need a much better cooling than normal ships, partly due to the requirement of keeping the Het alive.

Still doesn't seem to follow to me. Perhaps instead the characterization could be something along the lines of "While the cooling on a typical ship only needs to worry about keeping the crew comfortable in certain areas, the entirety of a Het ship needs to maintain a comfortable temperature for the Het inside to survive."